.github/workflows/test-zkvm-jolt.yml

```diff
@@ -0,0 +1,19 @@
+name: Test and clippy Jolt
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test:
+    uses: ./.github/workflows/test-zkvm.yml
+    permissions:
+      contents: read
+      packages: write
```

Comment:
Needed otherwise it'd complain `The nested job 'build_image' is requesting 'packages: write', but is only allowed 'packages: read'.`